tests/foreman/api/test_convert2rhel.py

```diff
@@ -70,6 +70,7 @@ def register_host(sat, act_key, module_org, module_loc, host, ubi=None):
         location=module_loc,
         insecure=True,
         repo=ubi,
+        force=True,
```

Comment:
Yes!

Reply:
@lhellebr I see @ogajduse complaint about force registration is valid that it might hide the actual issue underneath. Could you please answer why would we need force registration option with set to True by default. 

Force could be `False` by default and could be switched based on the condition!

Reply:
Me? I agree with Ondrej!